tasks/include/alternc-logos.yml

```diff
@@ -0,0 +1,72 @@
+---
+## AlternC logos and favicon customization
+
+- debug: 
+    msg: '{{ alternc_logo.local_file | splitext }}'
+    verbosity: 3
+
+- debug: 
+    msg: '{{ alternc_logo.local_file | splitext | first }}'
+    verbosity: 3
+
+- debug: 
+    msg: '{{ alternc_logo.local_file | splitext | last }}'
+    verbosity: 3
+
+- name: Resize logo (not .ico)
+  local_action: >
+    command convert {{ alternc_local_file_path }}/{{ alternc_logo.local_file }} 
+    -resize {{ alternc_logo.resize }}
+    {{ alternc_local_file_path }}/{{ alternc_logo.remote_file }}
+  become: no
+  when: alternc_logo.resize is string
+
+- name: Resize the several dimensions images for the favicon
+  local_action: >
+    command convert {{ alternc_local_file_path }}/{{ alternc_logo.local_file }} 
+    -resize  {{ logo_resize }}
+    {{ alternc_local_file_path }}/{{ alternc_logo.remote_file | splitext | first }}-{{ logo_resize }}.png
+  become: no
+  loop: '{{ alternc_logo.resize if alternc_logo.resize | type_debug == "list"  else "[]" }}'
+  loop_control:
+    loop_var: logo_resize
+
+- name: Build favicon file
+  local_action: >
+    command icotool -c -o  {{ alternc_local_file_path }}/{{ alternc_logo.remote_file }} 
```

Comment:
Yes of course. And the role tests it and eventually installs it, [here](https://github.com/UdelaRInterior/ansible-AlternC/blob/1890397900a5b7bac9526b0086ca791d58115ba5/tasks/10_checks-repos.yml#L39) (yes, yes, a lot of changes... :-)

Reply:
I'm using a Arch based distro. APT not work for me. You should check the OS.

Reply:
Or use a [generic module to manage packages](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html).

Reply:
The only problem may be the package name. But in these case, i ran:
```
$ pacman -S icoutils
```

Reply:
Indeed, in the code I pre-suppose local controller uses a debian like distro. The generic module  partially solves the problem, and we can add a comment when pausing: "... or either stop the playbook, install icoutils in your local controlle, and run the playbook again".

(When you agree with the answer and it is coded, please resolve conversation)

Reply:
Why "_The generic module partially solves the problem_"?

Reply:
I mean that it solves the problem if the controller's distribution is covered by the ansible generic package management module, the package exists in this distribution and has the same name. But we could hardly pretend to cover any OS of the controller in a single task. 

Maybe we could code an option to provide the `.ico` file instead of build it in the role from an image.

Reply:
If you set a variable with the package name?

Reply:
Yes, there are ways to deal with several distros in the same role. But I think it's over-coding to try to cover all the cases to the controller OS, we should focus our efforts on a proper and flexible installation of AlternC in the host. 

I think it's enough to try to cover as much cases as possible as far as it remains simple (your proposal to use [package module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html#ansible-collections-ansible-builtin-package-module) instead of `apt` is very good) and most of all propose workarounds to solve problems and extensions of the role: 
* propose to option to install `icotool` manually on the controller and re-run the role, 
* propose an option to provide to the role a manually built `.ico` file, instead of building it in the role from several sizes of an image, as it is now.

Reply:
Ok, I proposed my changes [here](https://github.com/UdelaRInterior/ansible-AlternC/pull/10)